action/steps/activate.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+# Run in a new empty directory
+ACTIVATE_LICENSE_PATH="$GITHUB_WORKSPACE/_activate-license"
+echo "Creating and changing to \"$ACTIVATE_LICENSE_PATH\" directory."
+mkdir -p "$ACTIVATE_LICENSE_PATH"
```

Comment:
I think it's better to move the variable and the creation and deletion for the directory outside of the steps themselves into [entrypoint](https://github.com/game-ci/unity-test-runner/blob/main/action/entrypoint.sh).

Variables will be passed, because we're using `source` to execute the steps.